assets/css/navigation.css

```diff
@@ -5,4 +5,98 @@
 
 .nav-wrapper .brand-logo {
     font-family: "KaushanScript", sans-serif;
-}
\ No newline at end of file
+}
+
+/* {
+    border:1px solid grey;
+    padding:10px;
+    height: 30px;
+} */
+
+ul li a{
+    display: flex;
+    text-align: center;
+}
+ul {
+    display: flex;
+    text-align: center;
+}
+
+ul li{
+    padding: 20px;
+}
+
+.icon{
+    padding-right: 10px;
+}
+
+.white ul a{
+    color: #000;
+}
+.white ul li.active {
+    background-color: #25adec; 
+    color:#fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li.active a{
+    background-color: #25adec; 
+    color: #fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li:hover a {
+    background-color: #73cdf7;
+    color:#fff;
+    border-radius: 5px 40px;
+}
+
+.white ul li:focus a {
```

Comment:
For now we are just working with the light mode, rename all of this to something more meaningful and appropriate rather than "white UI"